tool/generate-options.rb

```diff
@@ -19,8 +19,9 @@ def camelize(string)
 
 options_data = YAML.load_file('tool/options.yml')
 
-options = options_data.map do |constant, (name, type, default, *description)|
-  description = description.join(', ')
+options = options_data.map do |constant, ((name, *mri_names), type, default, description, *rest)|
+
+  raise "More than 5 arguments in #{constant}" unless rest.empty?
```

Comment:
"5 values" and I think it would be very helful full to show them (by keeping *values and do the splat assignment in the block)

Reply:
the key should be enough to find it, but ok I can print the values as well.

Reply:
The tricky case is having a `,` in the description and it's hard to spot in the .yml file, but much easier if the values are shown I think.